### PR TITLE
Add

### PR DESCRIPTION
**Pull request by beatriztruta**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by beatriztruta** [comment]
Book addition

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.
Please read docs/CONTRIBUTING.md guidelines!

File: books/free-programming-books-pt_BR.md [hunk 1]
```diff
@@ -14,6 +14,7 @@
 * [Dart](#dart)
 * [Docker](#docker)
 * [Engenharia de software](#engenharia-de-software)
+    * [Engenharia de software Moderna](#engenharia-de-software-moderna)
```
Comment (davorpa) [inline books/free-programming-books-pt_BR.md:17]: No new sections are needed

```suggestion
```

**Comment by beatriztruta** [comment]
Tirando erros

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**